src/Search_Replace_Command.php

```diff
@@ -328,7 +328,7 @@ private function php_handle_col( $col, $primary_keys, $table, $old, $new ) {
 		$replacer = new \WP_CLI\SearchReplacer( $old, $new, $this->recurse_objects, $this->regex, $this->regex_flags );
 
 		$where = $this->regex ? '' : " WHERE `$col`" . $wpdb->prepare( ' LIKE %s', '%' . self::esc_like( $old ) . '%' );
-		$primary_keys_sql = esc_sql( implode( ',', $primary_keys ) );
+		$primary_keys_sql = '`' . esc_sql( implode( '`,`', $primary_keys ) ) . '`';
```

Comment:
I don't think that the backticks count as quotes. Moreover, I suspect that the combination of `esc_sql()` with backticks is vulnerable to SQL injection _and_ will modify the keys so that they don't work (because of the added slashes).